hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/systemcontracts/CallAttemptHelpers.java

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
TY, I think this is a good solution for those unit tests.

Reply:
Thank you!